metron-platform/metron-management/README.md

```diff
@@ -54,9 +54,22 @@ The functions are split roughly into a few sections:
 * File functions - Functions around interacting with local or HDFS files
 * Configuration functions - Functions surrounding pulling and pushing configs from zookeeper
 * Parser functions - Functions surrounding adding, viewing, and removing Parser functions.
+* Indexing functions - Functions surrounding setting indexing parameters (batch size and timeout), setting the index, enabling/disabling index writer for sensors.
 * Enrichment functions - Functions surrounding adding, viewing and removing Stellar enrichments as well as managing batch size, batch timeout, and index names for the enrichment topology configuration
 * Threat Triage functions - Functions surrounding adding, viewing and removing threat triage functions.
 
+|  Type                     |  List of Functions                |                                     |                                                 |             |             |              | |
+| :-----------------------: | :-------------------------------: | :--------------------------------:  | :---------------------------------------------: | :---------: | :---------: | :----------: | :------------: |
+| Grok Functions            | [`GROK_EVAL`](#grok_eval)         | [`GROK_PREDICT`](#grok_predict)     | | | | | |
```

Comment:
I think the table of contents that you added is very helpful. I just don't like having to scroll horizontally to view them.

Would it make sense to do multiple tables that scroll vertically?  One for File Functions, another for Parser Functions, etc.  Or is there a down side to that approach that you are trying to avoid?

Reply:
Thanks @nickwallen for taking a look. I have reorganized vertically and split the tables in my latest commit. Do you think this is better?

Reply:
This is how it renders on HTML...

![image](https://user-images.githubusercontent.com/20395490/54606873-fcd42700-4a72-11e9-8b86-7b8bfe9a2575.png)

Reply:
That's definitely better (no horizontal scrolling), but I would still prefer just a simple vertical only list for all the functions.  That would match more closely what we have in stellar-common, rather than it being different here.  Or is there somewhere else that we take this approach?

I would certainly be open to being convinced.  Why do you prefer this approach?

Reply:
I want to make sure I understood you correctly. Are you suggesting a long vertical list of functions for the summary, like [here](https://github.com/apache/metron/tree/74ec7fedd48135123092b2a638ed1f2e2f57530c/metron-stellar/stellar-common#stellar-core-functions) ? In which case, we might not be able to add the function classification (Parser functions, Grok functions etc.). Or are you suggesting separate lists for each classification? Something like the below?

| **Grok functions** |
| Function 1 |
| Function 2 |
| .... |
| **Parser functions** |
| Function 1 |
| Function 2 |
| .... |

In stellar common, I found that the long list of vertical functions is a bit overwhelming for someone who wants a quick 'table of contents' for the list of functions. I created this summary so it will give me a quick snapshot of the available functions.

Reply:
How about 2 columns?: Left column has description once, right column has the functions, e.g.

 Type | Name
------|------
Grok Functions | ANAND_FTW
. | FUNCTION_2
. | FUNCTION_3
Parser Functions | PARSER_MAGIC

Reply:
This will appear something like this:

![image](https://user-images.githubusercontent.com/20395490/54623723-1afd4f80-4a92-11e9-9302-e3feb9c3748c.png)

We will not be able to merge the column names on the left due to limitations with github markdown. A stray empty row will be seen on the top, which is the unused heading row.

Reply:
That works for me.

Reply:
Works for me too. You might put "type" and "name" in the header row instead of empty cells.

Reply:
Done. Per latest feedback, this will look like:

![image](https://user-images.githubusercontent.com/20395490/54667675-eb425c00-4b12-11e9-8b39-da9912af6719.png)